app/v1/models/audit/DeductionsAuditDetail.scala

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package v1.models.audit
+
+import play.api.libs.functional.syntax._
+import play.api.libs.json.{JsPath, JsValue, OWrites}
+import v1.models.auth.UserDetails
+
+case class DeductionsAuditDetail(userType: String,
```

Comment:
Could you add some unit tests for this class to test the writes here like [this](https://github.com/hmrc/self-assessment-bsas-api/blob/master/test/v1/models/audit/GenericAuditDetailSpec.scala)